library/src/extra/rocsparse_csrgeam_numeric.cpp

```diff
@@ -22,9 +22,13 @@
  *
  * ************************************************************************ */
 
+#include <map>
```

Comment:
Let's try to separate runtime type dispatch from the implementation with separate files.